src/routes/Profile/Profile.tsx

```diff
@@ -176,8 +176,13 @@ const Profile: React.FunctionComponent = () => {
         ...p,
         ...getClaimButtonModesForVariants(claimedVariants, 'claimed'),
       }));
+      setClaimButtonBulkMode('claimed');
```

Comment:
```suggestion
      setClaimButtonBulkMode(claimedVariants.length === variants.length ? 'claimed' : 'claim');
```

we are not really sure if SDK is gonna claim all